tests/Unit/Ads/AccountServiceTest.php

```diff
@@ -461,6 +467,103 @@ public function test_get_billing_status_pending() {
 		);
 	}
 
+	public function test_get_ads_accoount_has_access() {
```

Comment:
Small typo in function name (same in the next few functions):
```suggestion
	public function test_get_ads_account_has_access() {
```

Also I can see that it's just referencing the function name it's testing, but it makes it easier if we describe what we are trying to test. Otherwise it would be helpful to add a docblock for the function to describe what it's testing. In this case it's testing that a Google Ads account does NOT have access, which you can't read from the function name.